test/test-RefractDataStructureTest.cc

```diff
@@ -2,16 +2,6 @@
 
 using namespace draftertest;
 
-static drafter::WrapperOptions MSONTestOptions(false, true);
```

Comment:
How come this was removed?

Reply:
it is moved into `test/draftertest.h`.
I made some refactoring of tests and remove restful parameter for macro and move similar macro definition into one file. But if you mean it will be better hold `TEST_MSON()` macro in this file, we can do it.